pallets/parachain-staking/src/lib.rs

```diff
@@ -393,6 +393,8 @@ pub mod pallet {
 		type MaxCollatorsPerNominator: Get<u32>;
 		/// Commission due to collators, set at genesis
 		type DefaultCollatorCommission: Get<Perbill>;
+		/// Percent of inflation set aside for parachain bond account
+		type ParachainBondReserveRatio: Get<Percent>;
```

Comment:
@crystalin should this be a mutable ratio, set by root? if so, I will move it to storage and the associated type will be the default value (so we don't have to add a field to genesis config).

Reply:
I think so